msg/en_US.utf8/utils.msg

```diff
@@ -1330,5 +1330,5 @@ valid options:\n\
                                   WARNING: This option has to be used with '-t(--transaction)' option.\n\
     -m, --module=TYPE             display detailed memory usage information of the specified TYPE.\n\
                                   TYPE of modules\n\
-                                  0 (or all)      all modules\n\
-                                  1 (or heap)     heap module\n
+                                  0 (or heap)     heap module\n\
+                                  1 (or all)      all modules\n
```

Comment:
what if the new module is added ?
is the new module appended to the end ?  
```
0 (or heap)
1 (or all)
2 (or new-module) ...
```

Reply:
`0 (or heap)
1 (or new-module)
2 (or all)`

Reply:
In my opinion, if we want to use 0 for the start, a fixed index should be for "all". I'd use -1 as the `diagdb` does. We should assume all options can be used in a script by a user. The user has to revise the script every time a module is added.